.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,44 @@
+# Thank you for contributing to iTwinUI by creating an issue!
+# Make sure you first do a search for existing issues so that this is is not a dulicate
+# Also visit our FAQ https://github.com/iTwin/iTwinUI/wiki/FAQ
```

Comment:
These comments are markdown-specific, so they are not appearing in the "rendered" template. Anything that needs to be exposed to the user should be added to the list under `body`.